src/files/files.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
separate the testing files into another folder

Reply:
done